app/controllers/discovered_hosts_controller.rb

```diff
@@ -199,15 +199,17 @@ def submit_multiple_auto_provision
 
   def setup_host_class_variables
     if @host.hostgroup
+      subnet = @host.hostgroup.subnet ? @host.hostgroup.subnet : @host.subnet
```

Comment:
Couldn't we just do `@host.hostgroup.subnet || @host.subnet`?

Reply:
Yep, and we can use @subnet below and do not need the helper vars. That makes this a little cleaner.

Reply:
Rebasing, I am leaving the helper vars as they are used down below.